README.md

```diff
@@ -756,8 +762,7 @@ async function unmark(datatype) {
 
 
 
-As a final step, run the `run.sh` command to ensure that the client code is rebuilt (once you stop
-the dev server, you won't be able to see your changes until the code is rebuilt):
+If you'd like to see the same changes show up on the original proxy, run the `run.sh` command to ensure that the client code is rebuilt (since we won't be skipping the client build this time):
```

Comment:
```suggestion
If you'd like to see the same changes show up on the original proxy (on port 8080), run the `run.sh` command to ensure that the client code is rebuilt (since we won't be skipping the client build this time):
```